app/src/main/java/com/teamfillin/fillin/presentation/map/SpaceDecoration.kt

```diff
@@ -0,0 +1,20 @@
+package com.teamfillin.fillin.presentation.map
+
+import android.graphics.Rect
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.*
+
+class SpaceDecoration(private val margin: Int) : ItemDecoration() {
+    override fun getItemOffsets(
+        outRect: Rect,
+        view: View,
+        parent: RecyclerView,
+        state: State
+    ) {
+        with(outRect) {
+            right = margin
+            bottom = margin
+        }
```

Comment:
```suggestion
        with(outRect) {
            right = margin.dp
            bottom = margin.dp
        }
```
ResolutionMetrics란  파일에 Number.dp라는 파일 보면 수치를 dp로 변환하는 getter가 있어서 그거 사용하면 디자인대로 spacing 적용 가능

Reply:
명강의 감사합니당